src/main/java/de/uni_kl/cs/discodnc/sinktree/Backlog_SinkTree.java

```diff
@@ -0,0 +1,39 @@
+package de.uni_kl.cs.discodnc.sinktree;
+
+import de.uni_kl.cs.discodnc.AnalysisConfig;
+import de.uni_kl.cs.discodnc.curves.ArrivalCurve;
+import de.uni_kl.cs.discodnc.curves.Curve;
+import de.uni_kl.cs.discodnc.network.server_graph.Flow;
+import de.uni_kl.cs.discodnc.network.server_graph.Server;
+import de.uni_kl.cs.discodnc.network.server_graph.ServerGraph;
+import de.uni_kl.cs.discodnc.network.server_graph.Turn;
+import de.uni_kl.cs.discodnc.sinktree.arrivalbounds.SinkTree_AffineCurves;
+
+public abstract class Backlog_SinkTree {
```

Comment:
If this is a util class please make it final. If it is not final you can create several instances via subclasses.